app/src/main/java/org/wikipedia/yearinreview/YearInReviewViewModel.kt

```diff
@@ -242,5 +244,23 @@ class YearInReviewViewModel() : ViewModel() {
         const val MIN_READING_ARTICLES = 5
         const val MIN_READING_MINUTES = 1
         const val MIN_READING_PATTERNS_ARTICLES = 5
+
+        // Whether Year-in-Review should be accessible at all.
+        // (different from the user enabling/disabling it in Settings.)
+        val isAccessible get(): Boolean {
+            if (Prefs.isShowDeveloperSettingsEnabled) {
```

Comment:
Notice that YiR is always accessible if Developer settings are enabled. I thought this would be more flexible than doing it by `isPreBetaRelease` etc.